### PR TITLE
basic python 3 support

### PR DESCRIPTION
**Pull request by mrpollo**
we are using the `future` package to get proper `Queue` support for both python 2.7 and > 3.0.

Thanks @Rosuav for the patch on #604

### CODE REVIEW

**Comment by hamishwillee** [comment]
It looks sane, but how is it tested?

**Comment by mrpollo** [comment]
@hamishwillee I'm tweaking our CI configs so they run it on both Python versions, no need for more tests, the idea is that we should be able to run the same code on both versions until we drop one of them (hopefully 2.7 in the future)

**Comment by hamishwillee** [comment]
OK. YOu'll have to re-merge. Perhaps @peterbarker should look at this too, but I'd be happy for you to self-merge :-)

**Comment by mrpollo** [comment]
going ahead and merging, travis fails on a pymavlink python3 known problem, this has been raised on https://github.com/ArduPilot/pymavlink/issues/4 and will soon be fixed, until then we can't really claim full python 3 support on dronekit

File: circle.yml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 machine:
+  post:
+    - pyenv global 2.7.10 3.5.0
```
Comment (cclauss) [inline circle.yml]: Why not the current 2.7.12 and 3.5.2?

File: dronekit/__init__.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
 import re
 from dronekit.util import errprinter
 from pymavlink import mavutil, mavwp
-from Queue import Queue, Empty
+from queue import Queue, Empty
```
Comment (cclauss) [inline dronekit/__init__.py]: Won't this break Python2 usage?

Normally, I would:

```python
try:                 # Python 3
    from queue import Queue, Empty
except ImportError:  # Python 2
    from Queue import Queue, Empty

```

**Pull request merged**